tests/functional/LoaderTest.php

```diff
@@ -200,4 +202,11 @@ protected function tearDown(): void
         chmod(self::DATABASE_PATH, 0644);
         unlink(self::DATABASE_PATH);
     }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)